pkg/core/native/management_test.go

```diff
@@ -21,7 +21,7 @@ func TestDeployGetUpdateDestroyContract(t *testing.T) {
 	ne, err := nef.NewFile(script)
 	require.NoError(t, err)
 	manif := manifest.NewManifest("Test")
-	require.NoError(t, err)
+	manif.Permissions = []manifest.Permission{}
```

Comment:
Because nil or empty permissions are serialised as an empty array (see line 83: https://github.com/neo-project/neo/pull/2254/files#diff-6a8fe6159fb062d52f619635b6c0e2123632e3a61acdc5f86b867da878af2e02R83). And several lines below we have manifests equality check. Without this line this check is failing.